.env

```diff
@@ -1 +1,10 @@
+
+# The fallback base url is configured to a public Spryker backend API 
+# to get you up and running fast. You can configure your custom backend, 
+# by setting the `SCOS_BASE_URL` value.
 ORYX_FALLBACK_SCOS_BASE_URL=https://glue.de.faas-suite-prod.cloud.spryker.toys
+
+# Using the `latest` feature flag will enable all the latest features. This might 
+# cause breaking changes, which why this flag is not enabled by default.
+#
+# ORYX_FEATURE_VERSION=latest
```

Comment:
But why it's commented?

Reply:
But I see 3 use-cases:
1. customers started with 1.0 and they are not merged latest changes from this repo
2. customers started with 1.0 and they merged latest changes from this repo
3. customers started with fresh new 1.1

So only 2nd group is affected and I'd say it kind of rare case, because most of our customers will start from scratch and with this setup we are locking them to the 1.0 version.

Reply:
if customer starts with 1.0, and they get this feature flag for free without opt-in explicitely, they will get breaking changes whenever they're updating their oryx dependencies. That's what I like to avoid. 
In CLI we can actually make it part of a question or template.

Reply:
As discussed, we set it to the latest version, which makes sense